tools/cosmovisor/CHANGELOG.md

```diff
@@ -36,6 +36,18 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## Features
+
+* [#19764](https://github.com/cosmos/cosmos-sdk/issues/19764) Use config file for cosmovisor configuration.
+
+## Improvements
+
+* [#19995](https://github.com/cosmos/cosmos-sdk/pull/19995):
```

Comment:
Nice description here, could you as well update the README with almost a copy paste of that :D

Reply:
I'll do this in a follow-up then, as I am touching cosmovisor. Merging now.